pkg/handler/rpms.go

```diff
@@ -11,17 +11,18 @@ import (
 	"github.com/openlyinc/pointy"
 )
 
-type RepositoryRpmHandler struct {
+type RpmHandler struct {
 	Dao dao.DaoRegistry
 }
 
-func RegisterRepositoryRpmRoutes(engine *echo.Group, rDao *dao.DaoRegistry) {
-	rh := RepositoryRpmHandler{
+func RepositoryRpmRoutes(engine *echo.Group, rDao *dao.DaoRegistry) {
```

Comment:
```suggestion
func RegisterRpmRoutes(engine *echo.Group, rDao *dao.DaoRegistry) {
```
did you mean to do this?

Reply:
I agree with the rename overall

Reply:
You renamed it from "RegisterRepositoryRpmRoutes" to "RepositoryRpmRoutes". Did you mean to rename it to "RegisterRpmRoutes"?

Reply:
ohhhhhh, good catch my braid was not picking up what you were getting at!